### PR TITLE
Enhancing the dataComments that are listed on the webpage

### PR DESCRIPTION
**Pull request by ruchiishahh**
hard-coded to 5 messages as of now, will be adding user input to change this.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -25,9 +25,11 @@ function getNameUsingArrowFunctions() {
  * Fetches stats from the servers and adds them to the DOM.
  */
 function getDataComment() {
-  fetch('/data').then(response => response.json()).then((dataComments) => {
+    fetch('/data').then(response => response.json()).then((dataComments) => {
       const commentContainer = document.getElementById("dataComment-container");
-      dataComments.forEach(dataComment => { 
+      var numOfCommentsToDisplay = 5;
```
Comment (bozhangbz) [inline portfolio/src/main/webapp/script.js:30]: Use const or let

Comment (ruchiishahh) [reply portfolio/src/main/webapp/script.js]: thanks, done!

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -44,6 +55,7 @@ function deleteDataComments(){
     }).then(getDataComments());
 }
 
+
```
Comment (chaconnewu) [inline portfolio/src/main/webapp/script.js:58]: nit: unnecessary

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -21,20 +21,31 @@ function getNameUsingArrowFunctions() {
...
           miniP.innerText = "Basketball Player: " + dataComment.message;
           commentContainer.appendChild(miniP);
       })
   });
 }
 
+/**
+ * Takes in user input's choice of number of comments to display.
+ */
+function getDiffNumberOfDataComments() {
```
Comment (chaconnewu) [inline portfolio/src/main/webapp/script.js:44]: looks like this function only resets the innerText and calls getDataComment(), which is unnecessary. There is a better way to organize the code. But for now, I think you can move the “innerText" line to getDataComment, and remove this function. Put getDataComment as the callback for input's onchange

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (ruchiishahh) [reply portfolio/src/main/webapp/script.js]: done!

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (ruchiishahh) [reply portfolio/src/main/webapp/script.js]: done, I've been trying to explore different ways to organize the code too - I made the change for now but will definitely come back to find the most optimal way.

**Pull request merged**